sdks/python/apache_beam/runners/portability/fn_api_runner/worker_handlers.py

```diff
@@ -614,8 +615,8 @@ def stop_worker(self):
     pass
 
   def host_from_worker(self):
-    # TODO(BEAM-8646): Reconcile the behavior on Windows platform.
-    if sys.platform == 'win32':
+    # TODO(BEAM-8646): Reconcile the across platforms.
```

Comment:
nit: extra 'the'.
Also, can you please comment on the bug which issues it causes on OS X?